components/form/CheckboxGroup.js

```diff
@@ -32,7 +32,7 @@ export function CheckboxGroup({ name, options }) {
         const optionValue = value || label;
         return <Checkbox key={index} name={name} value={optionValue} label={optionLabel} containerClassName="pb-1" />;
       })}
-      <ErrorMessage name={name} errorClassName="text-sm" />
+      <ErrorMessage name={name} className="text-sm" />
```

Comment:
The className is not needed since we have already defined it in the ErrorMessage component.

```suggestion
      <ErrorMessage name={name} />
```